Tests fail when using up-to-date version of maven-surefire-plugin

When running the tests using the current version 2.20.1 of the maven-surefire-plugin, tests are failing:
```
[INFO] Running com.cosylab.epics.caj.util.logging.test.ConsoleLogHandlerTest
2017-11-14T12:23:05.374 This is a simple message.
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.cosylab.epics.caj.util.logging.test.ConsoleLogHandlerTest
[INFO] Running com.cosylab.epics.caj.impl.handlers.test.AllTests
Channel Access Server in Java v1.1.D1
CLASS   : com.cosylab.epics.caj.cas.CAJServerContext
VERSION : Channel Access Server in Java [Java] v1.1.D1
SERVER : com.cosylab.epics.caj.cas.util.DefaultServerImpl
BEACON_ADDR_LIST :
AUTO_BEACON_ADDR_LIST : true
BEACON_PERIOD : 15.0
BEACON_PORT : 5065
SERVER_PORT : 5064
MAX_ARRAY_BYTES : 16384
IGNORE_ADDR_LIST:
STATE : INITIALIZED

[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.981 s <<< FAILURE! - in com.cosylab.epics.caj.impl.handlers.test.AllTests
[ERROR] testExceptionResponse(com.cosylab.epics.caj.impl.handlers.test.ExceptionResponseIT)  Time elapsed: 4.964 s  <<< ERROR!
gov.aps.jca.TimeoutException: pendIO timed out
        at com.cosylab.epics.caj.impl.handlers.test.ExceptionResponseIT.setUp(ExceptionResponseIT.java:86)

[INFO] Running com.cosylab.epics.caj.impl.test.AllTests
Channel Access Server in Java v1.1.D1
CLASS   : com.cosylab.epics.caj.cas.CAJServerContext
VERSION : Channel Access Server in Java [Java] v1.1.D1
SERVER : com.cosylab.epics.caj.cas.util.DefaultServerImpl
BEACON_ADDR_LIST :
AUTO_BEACON_ADDR_LIST : true
BEACON_PERIOD : 15.0
BEACON_PORT : 5065
SERVER_PORT : 5064
MAX_ARRAY_BYTES : 16384
IGNORE_ADDR_LIST:
STATE : INITIALIZED

Nov 14, 2017 12:23:17 PM com.cosylab.epics.caj.impl.CATransport processRead
SEVERE: Received payload size (65520) is larger than configured maximum array size (16384), disconnecting...
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 10.084 s <<< FAILURE! - in com.cosylab.epics.caj.impl.test.AllTests
[ERROR] testExtenderHeaderSupport(com.cosylab.epics.caj.impl.test.CATransportExtendedHeaderIT)  Time elapsed: 3.014 s  <<< FAILURE!
java.lang.AssertionError: expected:<gov.aps.jca.CAStatus[NORMAL=0,SUCCESS=1]=Normal successful completion> but was:<null>
        at com.cosylab.epics.caj.impl.test.CATransportExtendedHeaderIT.doGet(CATransportExtendedHeaderIT.java:86)
        at com.cosylab.epics.caj.impl.test.CATransportExtendedHeaderIT.testExtenderHeaderSupport(CATransportExtendedHeaderIT.java:103)


```

--- a/pom.xml
+++ b/pom.xml
@@ -130,6 +130,12 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.12.4</version>
                 <configuration>
+                    <systemProperties>
+                        <property>
+                            <name>java.util.logging.config.file</name>
+                            <value>test/resources/logging.properties</value>
+                        </property>
+                    </systemProperties>
                 </configuration>
             </plugin>
             <!-- Required add sources to deployment -->

--- a/test/resources/logging.properties
+++ b/test/resources/logging.properties
@@ -1,0 +1,2 @@
+# Low verbosity of logging for Maven tests.
+java.util.logging.ConsoleHandler.level = ERROR

